pgmpy/metrics/metrics.py

```diff
@@ -465,3 +465,126 @@ def SHD(true_model, est_model):
     shd = shd + (np.sum((d + d.T) > 0) / 2)
 
     return int(shd)
+
+
+def latent_admg(dag: DiscreteBayesianNetwork, observed: list) -> nx.MultiDiGraph:
```

Comment:
I would make this a private method as this won't have any application outside the compatibility score method. Rename this to `_latent_admg`.